docs/workflow_solutions/using_anaconda.md

```diff
@@ -260,6 +260,273 @@ Now we can be sure that the correct versions of the software will be installed o
     The example above is provided only for illustration purposes. The error has since been fixed, but the example above really happened and is helpful to explain version pinning.
 <!-- markdownlint-enable MD046 -->
 
+## Anaconda Environment Tutorial for PyTorch and TensorFlow
```

Comment:
Is all of this meant to be on this page? I see there is the tutorial in the tutorial section like we discussed. Maybe this needs to be edited? I also see the images are duplicated.

Reply:
I had not deleted the "Anaconda Environment Tutorial for PyTorch and TensorFlow" section from the using_anaconda.md page. I would delete this, and then make changes from your comments.